adam-core/src/main/scala/org/bdgenomics/adam/converters/VariantContextConverter.scala

```diff
@@ -787,7 +787,7 @@ class VariantContextConverter(
                                              gIndices: Array[Int]): Genotype.Builder = {
 
     // AD is an array type field
-    if (g.hasAD) {
+    if (g.hasAD && gIdx < g.getAD.size) {
```

Comment:
Does this only affect the AD field? I would've expected it to broadly impact fields that have the same array count as `AD`? Do we already check those?

Reply:
Yes, that file also has an `ADALL` VCF FORMAT Number=R Type=Integer genotype field that causes ArrayIndexOutOfBoundsExceptions, thus the other change at line 1289.

I'll add additional assertions to the test case to demonstrate this fix.

Reply:
Oh, hah! I'd missed that change earlier. Thanks for clarifying!